finder-app/Makefile

```diff
@@ -0,0 +1,25 @@
+# Compiler
+CC = gcc
+# Cross compiler variable
+CROSS_COMPILE ?=
+# Set target
+TARGET = writer
+# Set source
+SRCS = writer.c
+# Set object
+OBJS = writer.o
+
+# Default build
+default: $(TARGET)
+
+# Cross compile using aarch64-none-linux-gnu-gcc
+aarch64-none-linux-gnu-%:
+	$(MAKE) CROSS_COMPILE = aarch64-none-linux-gnu- $(TARGET)
```

Comment:
@tiba6275 I don't think you need this target, curious what this one is for.

Reply:
@dwalkes Was either left over from when I was trying calling the `$(TARGET)` rule from that rule and ended up misplaced, or I was thinking I needed it to call the default when calling `make` again from that rule. Which obviously doesn't make sense considering `TARGET` was already set and the default is the default. But I apparently also had spaces after the `CROSS_COMPILE` variable there so I don't think this rule was working anyway, will fix. The result of late-night coding.